app/_landing_pages/mesh.yaml

```diff
@@ -76,6 +86,28 @@ rows:
               cta:
                 text: Learn more
                 url: /mesh/vault/
+  - header:
+      type: h2
+      text: "Deployment modes"
+    columns:
+      - blocks:
+          - type: card
+            config:
+              title: Multi-zone
+              description: |
+                Group equivalent MeshServices across zones and expose a unified, zone-agnostic service with global failover capabilities.
+              cta:
+                text: Learn more
+                url: /mesh/mesh-multizone-service-deployment/
+      - blocks:
+          - type: card
+            config:
+              title: Single-zone
+              description: |
+                Run {{site.mesh_product_name}} in a single zone with a standalone Control Plane and interconnected Data Plane proxies.
+              cta:
+                text: Learn more
+                url: /mesh/single-zone/
   - header:
       type: h2
       text: "Policies"
```

Comment:
For the policies, I think these tiles should actually go away - unless we're specifically trying to call out "most popular policies". There's a mesh policies hub, similar to plugins: https://deploy-preview-1307--kongdeveloper.netlify.app/mesh/policies/ - we should add a prominent tile to that instead.